swagger/v1/openapi.json

```diff
@@ -572,87 +450,10 @@
         "responses": {
           "404": {
             "description": "profile not found",
-            "examples": {
-              "application/vnd.api+json": {
-                "errors": "Profile not found with ID invalid"
-              }
-            },
             "content": {}
           },
           "200": {
             "description": "retrieves a profile with included benchmark",
-            "examples": {
```

Comment:
It looks like you might still be running rswag with the --dry-run option on. To autogenerate examples, it has to not be `--dry-run`.

Reply:
I'm actually just running "rake rswag"  - in fact, these deletions seem to happen on master too. I think since on the refactoring no new `openapi.json` was committed, I do see the example for the 200 but not for the 404, let me check how can I get it back

Reply:
They were removed here https://github.com/RedHatInsights/compliance-backend/pull/492/files

Reply:
Oh I see the comment to run `/usr/local/bin/ruby -I/usr/local/bundle/gems/rspec-core-3.9.2/lib:/usr/local/bundle/gems/rspec-support-3.9.3/lib /usr/local/bundle/gems/rspec-core-3.9.2/exe/rspec --pattern spec/requests/\*\*/\*_spec.rb,\ spec/api/\*\*/\*_spec.rb,\ spec/integration/\*\*/\*_spec.rb --format Rswag::Specs::SwaggerFormatter --order defined
` - that does make a difference in the schema, but I don't see the examples in the Swagger UI just yet